src/background.js

```
@@ -82,27 +82,35 @@ browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
-                resolve()
-            } else if (new Date() - timeWas > timeoutMs) {
-                clearInterval(wait)
-                reject()
-            }
-        }, 20)
-    })
-}
+browser.runtime.onMessageExternal.addListener(async (request, sender, sendResponse) => {
+    switch (request.action) {
+        case 'addPersonalTheme':
+            const obj = request.obj
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)